build.gradle

```diff
@@ -60,3 +64,10 @@ private def projectVersion() {
 
     return tag
 }
+
+apply plugin: 'io.codearte.nexus-staging'
```

Comment:
should this go in `gradle/publish.gradle` or is it supposed to be applied to the root module?

Reply:
In the plugin docs they specifically mention than it should be applied to the root https://github.com/Codearte/gradle-nexus-staging-plugin#multi-project-build, it kinda makes sense but also is annoying